### ISSUE TITLE
Conv2d causes Process finished with exit code 139 (interrupted by signal 11: SIGSEGV)

### ISSUE THREAD

**Issue by Nikolai10** [tensorflow/tensorflow#24376]
**System information**
- Have I written custom code (as opposed to using a stock example script provided in TensorFlow): No
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): Ubuntu 18.04.1 LTS
- TensorFlow installed from (source or binary): source (conda)
- TensorFlow version (use command below): 1.12.0 
- Python version: 2.7
- CUDA/cuDNN version: 9.2/ 7.2.1
- GPU model and memory: RTX2070 8GB

**Describe the current behavior**

Process finished with exit code 139 (interrupted by signal 11: SIGSEGV);

**Describe the expected behavior**

Run a Convolutional Neural network with no error messages

**Code to reproduce the issue**
	# simplified example

	import tensorflow as tf
	mnist = tf.keras.datasets.mnist

	(x_train, y_train),(x_test, y_test) = mnist.load_data()
	x_train, x_test = x_train / 255.0, x_test / 255.0
	w, h = 28, 28

	# Reshape input data from (28, 28) to (28, 28, 1)
	x_train = x_train.reshape(x_train.shape[0], w, h, 1)
	x_test = x_test.reshape(x_test.shape[0], w, h, 1)

	# buggy example -> what is going on with conv2d?
	#==========================================================================
	model = tf.keras.models.Sequential([
	    tf.keras.layers.Conv2D(filters=64, kernel_size=2, padding='same', activation='relu', input_shape=(28,28,1)), # error
	    tf.keras.layers.Flatten(),
	    tf.keras.layers.Dense(512, activation=tf.nn.relu),
	    tf.keras.layers.Dropout(0.2),
	    tf.keras.layers.Dense(10, activation=tf.nn.softmax)
	])
	===========================================================================
	model.compile(optimizer='adam',
		      loss='sparse_categorical_crossentropy',
		      metrics=['accuracy'])

	model.fit(x_train, y_train, epochs=5)
	model.evaluate(x_test, y_test)

**Other info / logs**
The error appears, when Conv2D layer is used; if i uncomment likeso:

	   #tf.keras.layers.Conv2D(filters=64, kernel_size=2, padding='same', activation='relu', 

everythink works. 

More complicated examples like
https://colab.research.google.com/github/margaretmz/deep-learning/blob/master/fashion_mnist_keras.ipynb

fail with the same error message.

**Comment by ymodak** [comment]
I was able to run your code snippet successfully. I suspect you are running into memory issues. You can try running   
your script in [google colab](https://www.google.com/url?q=https://colab.sandbox.google.com&sa=D&source=hangouts&ust=1545167864524000&usg=AFQjCNEK4clc2sX5w9JnZ8OEsSOrM9WP2A) to confirm

**Comment by Nikolai10** [comment]
@ymodak thanks for your reply. I don't think this is a memory issue:

I was able to run the same code snippet (locally) using tensorflow-gpu==1.5.0 (conda). 
(i noticed that my CPU does not support AVX).

I guess i need to build from source, to run the latest tensorflow version?

**Comment by ymodak** [comment]
Glad it worked for you. You can also use prebuilt binaries for latest tensorflow apart from installing from sources.

**Comment by Nikolai10** [comment]
okay, thank you very much

**Comment by ymodak** [comment]
Are you satisfied with the resolution of your issue?
[Yes](https://goo.gl/forms/Oe0tEvODFRoI2gJF3)
[No](https://goo.gl/forms/fUjzOfrtkFbrOT8d2)
